flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -34,6 +34,23 @@ fi
 
 source "${END_TO_END_DIR}/test-scripts/test-runner-common.sh"
 
+# On Azure CI, set artifacts dir
+if [ ! -z "$TF_BUILD" ] ; then
+	export ARTIFACTS_DIR="${END_TO_END_DIR}/artifacts"
+	mkdir -p $ARTIFACTS_DIR || { echo "FAILURE: cannot create log directory '${ARTIFACTS_DIR}'." ; exit 1; }
+
+	# compress and register logs for publication on exit
+	function compress_logs {
+		echo "COMPRESSING build artifacts."
+		ARTIFACTS_FILE=${BUILD_BUILDNUMBER}.tgz
```

Comment:
can we name this `compressed_archive`? I keep getting confused and think this refers to a file within `ARTIFACTS_DIR`.

Reply:
good idea! I was also confused a few times.